liblogging.yaml

```diff
@@ -26,16 +24,16 @@ pipeline:
       repository: https://github.com/rsyslog/liblogging
       tag: v${{package.version}}
 
+  - uses: autoconf/configure
+
   - runs: |
-      libtoolize --copy
-      aclocal -I m4 --install
-      autoupdate
-      autoheader
-      autoconf
-      automake --foreign --add-missing --force-missing --copy
       ./configure --prefix=/usr --disable-journal --enable-stdlog
```

Comment:
We can remove this as it's handled by the configure pipeline above it

Reply:
Hmm, how to I add the specific --disable and --enable options?

Reply:
I figured it out!